style.css

```diff
@@ -0,0 +1,156 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* navbar section styling */
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 12px;
+  left: 0%;
+  right: 0%;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  position: absolute;
+  flex-wrap: wrap;
+  margin-right: 100px;
+}
+
+.navbar a {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 10px;
+  font-size: 19px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 10px;
+  text-decoration: none;
+}
+
+.navbar nav {
+  align-self: flex-end;
+}
+
+.navbar img {
+  display: flex;
+  right: 27.67%;
+  left: 16%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid #fff;
+  margin-bottom: 3px;
+  flex-grow: 0;
+  order: 0;
+  flex-direction: row;
+}
+
+#bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-self: flex-end;
+  margin: 2px 0 2px 10px;
+}
+
+#bar img {
+  display: flex;
+  right: 27%;
+  left: 16.67%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid white;
+  margin-bottom: 3px;
+}
+
+/* headline section styling */
+.headline {
+  overflow: hidden;
+  background-image: url(images/Group.png);
+  background-position-y: 125%;
+  background-color: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+  height: auto;
+  width: 100%;
+  padding: 160px 25px;
+  gap: 15px;
+  background-position: right bottom;
+  background-size: 145px;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.h1 {
+  color: #ff6b00;
+  width: 500px;
+  height: 5px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 210;
+  font-size: 44px;
+  line-height: 65px;
+  letter-spacing: 0.35px;
+  flex: none;
+}
+
+.h4 {
+  width: 325px;
+  height: 25px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 23px;
+  line-height: 30px;
+  align-items: center;
+  color: #fff5e1;
+  flex: none;
+  flex-grow: 0;
+  margin-top: 50px;
+  margin-bottom: 0;
```

Comment:
- [ ] [OPTIONAL] Kindly consider using the descriptive class selectors. This will help us to ensure that there are no other classes that inherit your stylings unknowingly ☺️.